### ISSUE TITLE
simple Output not working

### ISSUE THREAD

**Issue by razza-guhl** [MarcL/AmazonProductAPI#10]
Hi,

I tried the new version of AmazonProductAPI and it seems that the simple output is not working. 

Here is my sample code (almost same from your Readme), with a "var_dump" appended.

```
require('vendor/autoload.php');

use MarcL\AmazonAPI;
use MarcL\AmazonUrlBuilder;

// Keep these safe
$keyId = 'YOUR-AWS-KEY';
$secretKey = 'YOUR-AWS-SECRET-KEY';
$associateId = 'YOUR-AMAZON-ASSOCIATE-ID';

// Setup a new instance of the AmazonUrlBuilder with your keys
$urlBuilder = new AmazonUrlBuilder(
    $keyId,
    $secretKey,
    $associateId,
    'uk'
);

// Setup a new instance of the AmazonAPI and define the type of response
$amazonAPI = new AmazonAPI($urlBuilder, 'simple');

$items = $amazonAPI->ItemSearch('harry potter', 'Books', 'price');
echo "<pre>";
var_dump($items);
```
The Output look like this:

```
object(SimpleXMLElement)#6 (2) {
  ["OperationRequest"]=>
  object(SimpleXMLElement)#5 (3) {
[..]
```

So, this doesnt seem to be the 'simple' output format starting with "array".
I tried different transformation types, it always generates the same output.

Do i miss something?

Thank you for your great work!

**Comment by razza-guhl** [comment]
I might found the bug.
On Line 139 in src/AmazonAPI.php:

`return($parsedXml);`

should be:

`return $this->dataTransformer->execute($parsedXml);`

Then it worked for me

**Comment by MarcL** [comment]
Hey @razza-guhl,

Thanks. I must have missed that. I'll make a new release in a minute.

Cheers,
Marc

**Comment by MarcL** [comment]
I've just released v3.0.1 for you @razza-guhl with PR #11.
I need to write some tests to check this properly but done a quick fix for you for now.

Shout if you spot any more bugs or need any more functionality.

**Comment by razza-guhl** [comment]
Thank you for your work. Looks good!
